models/LineCont.yaml

```diff
@@ -0,0 +1,4 @@
+type: LineCont
+start: String
+indent: Boolean
```

Comment:
they cant be indented, nor nested. drop these properties that you dont use yet.

Reply:
done